update for SLH mcm script & language files

YanJunSunYJS opened the pull request:
I don't know if it can keep up with your development progress.

Added new translation text definitions to language files (refactored it and added translation descriptions), and modified mcm scripts accordingly

SkyrimLL commented:
Hi
Thanks for the contribution but I merged the changes to look at the new entries and I had to reverse the changes for now for multiple reasons:
- You made changes to the RUSSIAN text only
- All the other translation files were empty (0 size)
- You made changes to the SE version.. I am making updates to scripts in the LE version first and automatically copying changes to SE
- I made updates to the MCM file with new, untranslated entries. 

For these reasons, it was necessary to roll back the changes and try again later.
Please send me the new files in the LE branch of the mod if you want to give it another try. I updated it to the latest version.

YanJunSunYJS commented:
> Hi Thanks for the contribution but I merged the changes to look at the new entries and I had to reverse the changes for now for multiple reasons:
> 
> * You made changes to the RUSSIAN text only
> * All the other translation files were empty (0 size)
> * You made changes to the SE version.. I am making updates to scripts in the LE version first and automatically copying changes to SE
> * I made updates to the MCM file with new, untranslated entries.
> 
> For these reasons, it was necessary to roll back the changes and try again later. Please send me the new files in the LE branch of the mod if you want to give it another try. I updated it to the latest version.

Fixed :)
